### DISCUSSION TITLE
Why is Bevy called Bevy?

### DISCUSSION THREAD

**Discussion by amunrarara** [bevyengine/bevy#10114] [General]
What's the story there?

**Comment by DGriffin91** [comment]
IIRC cart talks a bit about it on this podcast episode! https://rustgamedev.com/episodes/interview-with-carter-anderson-bevy

**Reply by IceSentry** [reply]
It's also mentioned at the end of the introduction blog post
https://bevyengine.org/news/introducing-bevy/#join-the-bevy

> I want Bevy to become a vibrant developer community ... thats actually why I chose the name! A Bevy is a group of birds, just like we are a group of game developers. Join the Bevy!

**Reply by DGriffin91** [reply]
But you'll have to listen to the podcast to find out things like *when* and *what* was the first bevy. (hint, it's not this game engine)

**Reply by amunrarara** [reply]
Ahhhh, you've kickstarted my memory into life again. It's also right here, first page of [the Bevy Book](https://bevyengine.org/learn/book/introduction/): 

> What's a BEVY?
> A bevy is a group of birds!
> But Bevy is also a refreshingly simple data-driven game engine built in Rust. It is [free and open-source](https://github.com/bevyengine/bevy) forever.

Awesome. Thanks for the info! I'm darn glad to be a bird in the bevy!
